sys-process/lsof/lsof-4.98.0.ebuild

```diff
@@ -28,6 +28,15 @@ BDEPEND="
 # Needs fixing first for sandbox
 RESTRICT="test"
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-4.98.0-fix-clang-version-parser.patch
+)
+
+src_prepare() {
+	default
+	eautoreconf
+}
```

Comment:
Add a comment above the eautoreconf noting that it's just there for the patch, so we know to drop it later.

Reply:
Done